docs/IntegrationTest.md

```diff
@@ -0,0 +1,191 @@
+    ```
+
+1. Run the command provided in the output to print your jenkins password to the Cloud Shell console.
+You will need this to log in, and a new password is created whenever you run `helm install` or
+`helm upgrade`.
+
+### Checking Jenkins status
+1. Run the following to check the [status of your certificate](
+https://cloud.google.com/load-balancing/docs/ssl-certificates#certificate-resource-status):
+    ```bash
+    gcloud beta compute ssl-certificates describe $CERTIFICATE_NAME
+    ```
+
+    You should see the status as `ACTIVE`. This may take up to an hour after the first helm install.
+
+1. Run the following to check the status of the Kubernetes objects involved with running Jenkins on
+your cluster.
+    ```bash
+    kubectl describe deployment $JENKINS_NAME
+    kubectl describe ingress $JENKINS_NAME
+    ```
+
+    Here's what that should look like (`#` can be any number, including hex digits):
+    
+    ```
+    $ user@cloudshell:~ (exampleproject)$ kubectl describe deployment $JENKINS_NAME
+    Name:               jenkins-test-host
+    Namespace:          default
+    CreationTimestamp:  ...
+    Labels:             chart=jenkins-0.32.9
+                        component=jenkins-test-host-jenkins-master
+                        heritage=Tiller
+                        release=jenkins-test-host
+    ...
+    ...
+    ...
+    
+    $ user@cloudshell:~ (exampleproject)$ kubectl describe ingress $JENKINS_NAME
+    Name:             jenkins-test-host
+    Namespace:        default
+    Address:          #.#.#.#
+    Default backend:  default-http-backend:80 (#.#.#.#:8080)
+    Rules:
+      Host                              Path  Backends
+      ----                              ----  --------
+      example.com
+                                           jenkins-test-host:8080 (<none>)
+    Annotations:
+      ingress.kubernetes.io/url-map:                k8s-um-default-jenkins-test-host--7f##############
+      kubernetes.io/ingress.allow-http:             false
+      kubernetes.io/ingress.global-static-ip-name:  jenkins-address
+      ingress.gcp.kubernetes.io/pre-shared-cert:    jenkins-host-certificate
+      ingress.kubernetes.io/backends:               {"k8s-be-#####--7f##############":"HEALTHY","k8s-be-#####--7f##############":"HEALTHY"}
+      ingress.kubernetes.io/https-forwarding-rule:  k8s-fws-default-jenkins-test-host--7f##############
+      ingress.kubernetes.io/https-target-proxy:     k8s-tps-default-jenkins-test-host--7f##############
+      ingress.kubernetes.io/ssl-cert:               jenkins-host-certificate
+    ```
+
+### Testing the plugin on Jenkins
+1. Make sure you have another cluster and service account set up for testing deployments as
+described in the GKE Plugin [usage documentation](Home.md#usage). You will need to repeat steps 3-8 
+for the [IAM Credentials](Home.md#iam-credentials) section on this new Jenkins instance. This is the
```

Comment:
It may make sense for a future PR to create a separate helm chart for setting up the necessary cluster role bindings for the GCP SA, that way we can automate away these manual steps.

Reply:
When we've refactored the Google oauth plugin to enable the pipeline we should be able to do this through the existing helm chart's Configuration as Code values

Reply:
Sounds good.

Reply:
Added a TODO for this future change